src/libsyntax/parse/parser.rs

```diff
@@ -81,6 +81,8 @@ type ItemInfo = (Ident, ItemKind, Option<Vec<Attribute> >);
 pub enum PathParsingMode {
     /// A path with no type parameters; e.g. `foo::bar::Baz`
     NoTypesAllowed,
```

Comment:
I removed all existing uses in https://github.com/rust-lang/rust/pull/33041, but a new one is introduced in https://github.com/rust-lang/rust/pull/32875
Merging these two variants into one with `ImportPrefix`'s semantics does no harm, I guess. I'll do it as a part of https://github.com/rust-lang/rust/pull/33041